Packs/CofenseTriage/Integrations/CofenseTriagev2/CofenseTriagev2.py

```diff
@@ -741,28 +741,30 @@ def get_report_png_by_id(triage_instance, report_id):
     ).content
 
 
+def build_triage_instance():
+    demisto_params = {
+        "start_date": parse_date_range(demisto.getParam("date_range"))[0].isoformat(),
+        "max_fetch": int(demisto.getParam("max_fetch")),
```

Comment:
this should be in try-except clause in case the user has entered a non-numeric value

Reply:
Any exceptions during initialization will be caught by the `except Exception` at the bottom of `main()`. Is that sufficient?

Reply:
It is, but its better to inform the user with an dedicated error msg, indicating the arg/param name and what value is expected. It's up to your decision.